server/test/services/geo/esri/EsriClientTest.java

```diff
@@ -177,4 +182,83 @@ public void getAddressSuggestionsWithError() throws Exception {
     assertThat(group.getWellKnownId()).isEqualTo(0);
     assertThat(group.getOriginalAddress()).isEqualTo(address);
   }
+
```

Comment:
**suggestion:** Could you add a test where both Region and RegionAbbr are both > 2 letters?